modules/virt-configuring-openshift-root-CA-for-providers.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * virt/live_migration/virt-about-mtv-providers.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-configuring-openshift-root-CA-for-providers_{context}"]
+= Configuring the OpenShift root certificate authority for providers
+
+You must configure an {product-title} provider for each cluster that you are including in the migration, and each provider requires a certificate authority (CA) for the cluster. It is important to configure the root CA for the entire cluster to avoid a CA's early expiration, which causes the provider to fail.
+
+.Procedure
+
+* Run the following command against the cluster for which you are creating the provider:
++
+[source,terminal]
+----
+$ oc get cm kube-root-ca.crt -o=jsonpath={.data.ca\\.crt}
+----
+
```

Comment:
Maybe we should mention that the output from that command, needs to be pasted into the `CA Certificate` field of the provider creation form.

It will look like this, one or more begin/end certificates 
```
-----BEGIN CERTIFICATE-----
...
-----END CERTIFICATE-----
```

Reply:
@awels, I've added this info, but I wasn't sure what to call the output that's pasted. I did some research and came up with <CA_certificate_content>. Is that accurate?